worlds/dark_souls_3/__init__.py

```diff
@@ -1457,19 +1458,19 @@ def smooth_items(item_order: List[Union[DS3ItemData, DarkSouls3Item]]) -> None:
                         f"contain smoothed items, but only {len(converted_item_order)} items to smooth."
                     )
 
+                sorted_spheres = []
                 for sphere in locations_by_sphere:
                     locations = [loc for loc in sphere if loc.item.name in names]
 
                     # Check the game, not the player, because we know how to sort within regions for DS3
                     offworld = ds3_world._shuffle([loc for loc in locations if loc.game != "Dark Souls III"])
                     onworld = sorted((loc for loc in locations if loc.game == "Dark Souls III"),
                                      key=lambda loc: loc.data.region_value)
+                    sorted_spheres.extend(onworld + offworld)
```

Comment:
```suggestion
                    sorted_spheres.extend(onworld)
                    sorted_spheres.extend(offworld)
```
A nested for loop or itertools.chain could have been used in the original code to avoid concatenating to create a new list. With the PR changes, the concatenation is simpler to avoid and remains easily readable.